### PR TITLE
[13.0] shopfloor_mobile: Add setting page improvements

### PR DESCRIPTION
**Pull request by TDu**
Move the log out button to the main setting page.
On the profile and language setting buttons, add the
active value.

### CODE REVIEW

File: shopfloor_mobile/static/wms/src/settings/settings.js [hunk 1]
```diff
@@ -6,7 +6,28 @@
 
 export var SettingsControlPanel = Vue.component("settings-control-panel", {
     data: function() {
-        return {};
+        return {
+            // Why do I have to do the sub 1 to get the correct name ?
+            profile_name: this.$root.has_profile
+                ? this.$root.profiles[this.$root.profile.id - 1].name
+                : "?",
```
Comment (simahawk) [inline shopfloor_mobile/static/wms/src/settings/settings.js]: why can't you use `this.$root.profile.name`?

File: shopfloor_mobile/static/wms/src/settings/settings.js [hunk 2]
```diff
@@ -6,7 +6,28 @@
...
+        logout: function() {
+            this.$root.logout();
+        },
+    },
+    computed: {
+        available_languages() {
+            return this.$root.available_languages;
+        },
+        active_language() {
+            var language = this.available_languages.find(
```
Comment (simahawk) [inline shopfloor_mobile/static/wms/src/settings/settings.js]:
```suggestion
            const language = this.available_languages.find(
```

File: shopfloor_mobile/static/wms/src/settings/settings.js [hunk 3]
```diff
@@ -15,18 +36,23 @@ export var SettingsControlPanel = Vue.component("settings-control-panel", {
...
                         <v-btn @click="$router.push({'name': 'language'})">
                             <v-icon>mdi-flag</v-icon>
-                            <span>{{ $t("screen.settings.language.name") }}</span>
+                            <span>{{ $t("screen.settings.language.name") }} - {{ this.active_language }}</span>
                         </v-btn>
                     </v-col>
                 </v-row>
+                <v-row align="center">
+                    <v-col class="text-center" cols="12">
+                        <btn-action @click="logout()">{{ $t('screen.settings.profile.action.logout') }}</btn-action>
```
Comment (simahawk) [inline shopfloor_mobile/static/wms/src/settings/settings.js]:
```suggestion
                        <btn-action @click="logout()">{{ $t('app.action.logout') }}</btn-action>
```
and move the translation in i18n file as this is a generic action that could be done from everywhere.

**Review by simahawk — CHANGES_REQUESTED** [review]

File: shopfloor_mobile/static/wms/src/settings/settings.js (hunk 1 above)
Comment (TDu) [reply shopfloor_mobile/static/wms/src/settings/settings.js]: Doh, changed

File: shopfloor_mobile/static/wms/src/settings/settings.js (hunk 2 above)
Comment (TDu) [reply shopfloor_mobile/static/wms/src/settings/settings.js]: Done

File: shopfloor_mobile/static/wms/src/settings/settings.js (hunk 3 above)
Comment (TDu) [reply shopfloor_mobile/static/wms/src/settings/settings.js]: Moved

File: shopfloor_mobile/static/wms/src/settings/settings.js [hunk 4]
```diff
@@ -6,7 +6,25 @@
...
+        },
+    },
+    computed: {
+        available_languages() {
+            return this.$root.available_languages;
+        },
+        active_language() {
+            const language = this.available_languages.find(
+                ({id}) => id === this.$i18n.locale
+            );
```
Comment (guewen) [inline shopfloor_mobile/static/wms/src/settings/settings.js:25]: Can we use es6?

Comment (guewen) [reply shopfloor_mobile/static/wms/src/settings/settings.js:25]: I see @simahawk proposed the change to `const` so I suppose yes

Comment (TDu) [reply shopfloor_mobile/static/wms/src/settings/settings.js:25]: Yes es6 is being used in the project

**Review by simahawk — APPROVED** [review]

**Comment by TDu** [comment]
Squashed

**Review by guewen — APPROVED** [review]

**Pull request merged**